### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by ericpre**
Add documentation to describe how to download, install and use the hyperspy-bundle. This is mostly adapted from previous workshop with some improvement and updates.

@francisco-dlp, @jlaehne, do you want to review this PR?

The documentation is available at https://hyperspy.org/hyperspy-bundle

### CODE REVIEW

**Comment by hakonanes** [comment]
The docs are very nice and tidy, I wish we had those to point participants at a diffraction workshop at NTNU in June!

Two comments:
1. The [QtConsole image](https://hyperspy-bundle.readthedocs.io/en/latest/verify.html#jupyter-qtconsole) has tiny text which is only readable when opening the image separately
2. How about adding "HyperSpy Bundle" beside the HyperSpy logo in the navigation bar ([relevant config](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/configuring.html#add-a-logo-title))?

**Comment by hakonanes** [comment]
Also, it should be "listed below" here: https://hyperspy-bundle.readthedocs.io/en/latest/index.html#difference-with-winpython.

File: docs/download.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Depending on your computer, choose the most suitable installer:
+
+.. figure:: _static/bundle_release_download.png
+   :width: 100 %
+   :target: https://github.com/hyperspy/hyperspy-bundle/releases/latest
+   :alt: Links HyperSpy bundle downloads
+   :figwidth: 80%
+
+   Select a suitable installer depending on what type of processor is in your computer:
+   on Windows and Linux, one of either Intel or AMD on Mac, one of either Intel of M1.
```
Comment (jlaehne) [inline docs/download.rst:28]:
```suggestion
   on Windows and Linux, one of either Intel or AMD, and on Mac, one of either Intel of M1.
```

File: docs/download.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            href="https://github.com/hyperspy/hyperspy-bundle/releases/latest">
+                Download HyperSpy-bundle installer
+        </a>
+    </div>
+    <br>
+
+
+Depending on your computer, choose the most suitable installer:
+
+.. figure:: _static/bundle_release_download.png
```
Comment (jlaehne) [inline docs/download.rst:21]: I'll send a PR to correct `RELEASE_TEXT.md`. This figure would need to be updated accordingly.

Comment (jlaehne) [reply docs/download.rst:21]: See #68

**Comment by ericpre** [comment]
Thanks @hakonanes  and @jlaehne for reviewing.
The image of the terminal prompt still need to be updated, but I can't do it right now.

**Comment by jlaehne** [comment]
> The image of the terminal prompt still need to be updated, but I can't do it right now.

Couldn't you just increase the `:figwidth:` to 70% or 80%?

Also the figure `bundle_release_download.png` will need to be updated due to typos ... but I guess there first needs to be a new release before a new screenshot can be made.

**Review by jlaehne — APPROVED** [review]

**Pull request merged**